server/routes/journeys/update-referral/additional-information/controller.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { MESSAGE_REFERRAL_ADDITIONAL_INFO_UPDATED, PatchReferralController } from '../../base/patchReferralController'
+import { generateSaveTimestamp, getMaxCharsAndThresholdForAppend } from '../../../../utils/appendFieldUtils'
+
+export class UpdateAdditionalInfoController extends PatchReferralController {
+  GET = async (req: Request, res: Response) => {
+    res.render('referral/additional-information/view', {
+      isProactiveReferral: req.journeyData.referral!.isProactiveReferral,
```

Comment:
i don't think this property is need for the view file

Reply:
True, removed